tensorflow/python/estimator/estimator.py

```diff
@@ -64,6 +64,12 @@
     ['features', 'labels', 'mode', 'params', 'self', 'config'])
 
 
+def _check_string_or_not(name):
```

Comment:
Because you only use this in one place, can you

- make this function local to where you use it
- make the error message more specific (specifically, the string is a variable name)

Reply:
Sure, I will fix the this one.